src/Model/Resource.php

```diff
@@ -30,7 +30,7 @@ class Resource implements EntityInterface, InvalidPropertyInterface
      * @param array $properties hash of properties to set in this resource
      * @param array $options list of options to use when creating this resource
      */
-    public function __construct(array $properties = [], array $options = [])
+    final public function __construct(array $properties = [], array $options = [])
```

Comment:
```suggestion
    public function __construct(array $properties = [], array $options = [])
```

Reply:
See the comment of Endpoint, I would like to ask you the same question here.

Reply:
Solved this by adding a psalm constraint, as dictated by psalm. This solution thus only works if any project dependent on Muffin also uses psalm.